packages/components/src/navigation/index.js

```diff
@@ -83,30 +89,36 @@ const Navigation = ( { activeItemId, children, data, rootTitle } ) => {
 		);
 	};
 
+	const render = ( { className: animateClassName } ) => (
+		<div
+			className={ classnames(
+				'components-navigation__level',
+				animateClassName
+			) }
+		>
+			{ children( {
+				level,
+				NavigationBackButton,
+				parentLevel,
+			} ) }
+		</div>
+	);
+
 	return (
 		<Root className="components-navigation">
-			<Animate
-				key={ level.id }
-				type="slide-in"
-				options={ {
-					origin: isNavigatingBack ? 'right' : 'left',
-				} }
-			>
-				{ ( { className: animateClassName } ) => (
-					<div
-						className={ classnames(
-							'components-navigation__level',
-							animateClassName
-						) }
-					>
-						{ children( {
-							level,
-							NavigationBackButton,
-							parentLevel,
-						} ) }
-					</div>
-				) }
-			</Animate>
+			{ onMount ? (
+				render( {} )
```

Comment:
In my testing, this never gets called.  I think we'll need to use state or a ref to track the mounted state.

`const isMounted = useRef(true)`